src/scripts/test_all_setup.sh

```
@@ -117,17 +117,20 @@ start_attention_broker(){
-        --name="mork-test-server" \
+        --name="${name}" \
         --network host \
         -e MORK_SERVER_ADDR=0.0.0.0 \
-        -e MORK_SERVER_PORT=40022 \
-        "${MORK_SERVER_IMAGE}" "$@"
+        -e MORK_SERVER_PORT=${port} \
+        "${MORK_SERVER_IMAGE}"
 
-    until curl --silent http://localhost:${DAS_MORK_PORT}/status/-; do
-    echo "[SETUP-INFO] Waiting for MORK..."
+    until curl --silent http://localhost:${port}/status/-; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)